### PR TITLE
[simple_kit]: KeyboardAmount and KeyboardPin

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }) : super(key: key);
+
+  final Color? color;
+  final String assetName;
+
+  @override
+  Widget build(BuildContext context) {
+    return ConstrainedBox(
+      constraints: BoxConstraints(
+        maxWidth: 102.r,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart:19]: .w?

File: plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        maxHeight: 56.r,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart:20]: .h?

File: plugins/simple_kit/lib/src/keyboards/components/numeric_keyboard_preset.dart [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      onTap: onTap,
+      child: Container(
+        width: 102.w,
+        height: 48.h,
+        decoration: BoxDecoration(
+          color: selected ? SColorsLight().grey5 : SColorsLight().white,
+          borderRadius: BorderRadius.circular(24.r),
+          border: selected
+              ? Border.all(
+                  width: 2.w,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/keyboards/components/numeric_keyboard_preset.dart:30]: .r?

File: plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart (hunk 1 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart]: done

File: plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart (hunk 2 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/icons/102x56/base/simple_base_svg_102x56.dart]: done

File: plugins/simple_kit/lib/src/keyboards/components/numeric_keyboard_preset.dart (hunk 3 above)
Comment (eli1stark) [reply plugins/simple_kit/lib/src/keyboards/components/numeric_keyboard_preset.dart]: done

**Pull request merged**